vars/network_vars.yml

```diff
@@ -38,7 +38,7 @@ br0vlan_systemd_networkd_network:
         - Name: "{{ network_interface }}"
     - Network: "{{ [ {'Bridge': 'br0'} ] + extra_network_config | default([]) }}"
     - BridgeVLAN:
-        - VLAN: 159
+        - VLAN: "{{ br0vlan | default('') }}"
         - PVID: 4094
```

Comment:
There are the same issue PVID and EgressUntagged.

Reply:
I'm going to remove the whole section